packages/edit-site/src/components/template-actions/index.js

```diff
@@ -82,48 +121,26 @@ export default function TemplateActions( {
 			createErrorNotice( errorMessage, { type: 'snackbar' } );
 		}
 	}
-
 	return (
-		<DropdownMenu
-			icon={ moreVertical }
-			label={ __( 'Actions' ) }
-			className={ className }
-			toggleProps={ toggleProps }
-		>
-			{ ( { onClose } ) => (
-				<MenuGroup>
-					{ isRemovable && (
-						<>
-							<RenameMenuItem
-								template={ template }
-								onClose={ onClose }
-							/>
-							<DeleteMenuItem
-								onRemove={ () => {
-									removeTemplate( template );
-									onRemove?.();
-									onClose();
-								} }
-								title={ template.title.rendered }
-							/>
-						</>
-					) }
-					{ isRevertable && (
-						<MenuItem
-							info={ __(
-								'Use the template as supplied by the theme.'
-							) }
-							onClick={ () => {
-								revertAndSaveTemplate();
-								onClose();
-							} }
-						>
-							{ __( 'Clear customizations' ) }
-						</MenuItem>
-					) }
-				</MenuGroup>
-			) }
-		</DropdownMenu>
+		<>
+			<MenuItem
+				info={ __( 'Use the template as supplied by the theme.' ) }
```

Comment:
This text is also showing for template parts. Should we have a conditional message and show "Use the template-part as supplied by the theme."?

Reply:
Let's do that in a follow up that will update the confirmation modal for this.